modules/nf-core/limma/differential/templates/limma_de.R

```diff
@@ -353,6 +367,8 @@ if (! is.null(opt\$block)){
 }
 if (! is.null(opt\$correlation)){
     lmfit_args[['correlation']] <- as.numeric(opt\$correlation)
+} else if (!is.null(opt\$mixed_model) && opt\$mixed_model) {
+    lmfit_args[['correlation']] <- corfit\$consensus.correlation
```

Comment:
```suggestion
    lmfit_args[['correlation']] <- corfit\$consensus.correlation
    lmfit_args[['block']] <- sample.sheet[[opt\$block]]
```

We should pass the `block` to lmFit in the same way as to duplicateCorrelation(), right? Also, maybe remove the existing `block` conditional above.

Reply:
Well one can pass correlation value form via opt\$correlation what I think should have higher priority. With that in mind I don't think that we can merge it in a clean way and keep this flexibility.